cmd/neofs-cli/modules/object/util.go

```diff
@@ -81,3 +86,89 @@ func readOID(cmd *cobra.Command, id *oid.ID) {
 	err := id.DecodeString(cmd.Flag("oid").Value.String())
 	common.ExitOnErr(cmd, "decode object ID string: %w", err)
 }
+
+// common interface of object operation's input which supports sessions.
+// Implemented on types like internal.PutObjectPrm.
+type sessionPrm interface {
+	SetSessionToken(*session.Object)
+}
+
+// forwards all parameters to _readSession and object as nil.
+func readSessionGlobal(cmd *cobra.Command, dst sessionPrm, key *ecdsa.PrivateKey, cnr cid.ID) {
+	_readSession(cmd, dst, key, cnr, nil)
+}
+
+// forwards all parameters to _readSession.
+func readSession(cmd *cobra.Command, dst sessionPrm, key *ecdsa.PrivateKey, cnr cid.ID, obj oid.ID) {
+	_readSession(cmd, dst, key, cnr, &obj)
+}
+
+// decodes object session from JSON file from "session" command flag if it is provided,
+// and writes resulting session into the provided sessionPrm. Checks:
+//
+//	if session verb corresponds to given sessionPrm according to its type
+//	relation to the given container
+//	relation to the given object if non-nil
+//	relation to the given private key used withing the command
+//	session signature
```

Comment:
isn't it hard to read?

Reply:
How can I improve readability?

Reply:
we usually use `-` in such cases. it is hard for me to understand where the sentence ends

lets discuss

Reply:
Agree with message excretion, added dashes.